### PR TITLE
[infra] Add man page for onecc

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit adds man page for onecc tool.

Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

**Comment by jyoungyun** [comment]
This file will be shown like this:
_It ignores fonts format like bold and italic._
```
ONECC(1)                               User Commands                              ONECC(1)

NAME
       onecc - run ONE driver via several commands or configuration file

SYNOPSIS
       onecc [-h] [-v] [-C CONFIG] [COMMAND <args>]

DESCRIPTION
       onecc is a command line tool to execute ONE driver via several commands or configu‐
       ration file.

   Configuration file:
       onecc takes input as a configuration file that supports ini format.   A  configura‐
       tion  file  consists  of sections, each led by a [section] header.  Each section is
       the ONE driver you want to run, and consists of commands in a key/value combination
       to pass to the driver.

OPTIONS
       -h, --help
              show this help message and exit

       -v, --version
              show program's version number and exit

       -V, --verbose
              output additional information to stdout or stderr

       -C CONFIG, --config CONFIG
              run with configuation file

   compile to circle model
       import Convert given model to circle. See one-import(1) for details.

       optimize
              Optimize circle model. See one-optimize(1) for details.

       quantize
              Quantize circle model. See one-quantize(1) for details.

   package circle model
       pack   Package circle and metadata into nnpackage. See one-pack(1) for details.

   run backend tools
       codegen
              Code generation tool. See one-codegen(1) for details.

EXAMPLES
   Use command line interface
       onecc  import  tf --v1 -i input path -o output path -I input arrays -s input shapes
       -O output arrays
              import tf model

       onecc import tflite -i input path -o output path
              import tflite model

       onecc import onnx -i input path -o output path
              import onnx model

       onecc optimize -i input path -o output path optimize arguments
              optimize circle model

       onecc quantize -i input path -o output path -d input data
              quantize circle model

       onecc pack -i input path -o output path
              package circle and metadata into nnpackage

       onecc codegen -b backend -- backends arguments
              generate backend code

   Use configuration file
       The configuration file should be written in the following format:

              [onecc]
              one-import-tf=True
              one-import-tflite=False
              one-import-bcq=False
              one-import-onnx=False
              one-optimize=True
              one-quantize=True
              one-pack=True
              one-codegen=True

              [one-import-tf]
              input_path=/path/to/inception_v3.pb
              output_path=inception_v3.circle
              input_arrays=input
              input_shapes=1,299,299,3
              output_arrays=InceptionV3/Predictions/Reshape_1
              converter_version=v1
              model_format=graph_def

              [one-optimize]
              input_path=inception_v3.circle
              output_path=inception_v3.opt.circle
              generate_profile_data=False

              [one-quantize]
              input_path=inception_v3.opt.circle
              output_path=inception_v3.quantized.circle
              input_data=inception_v3_test_data.h5

              [one-pack]
              input_path=inception_v3.quantized.circle
              output_path=inception_v3_pack

              [one-codegen]
              backend=dummy
              command=-o sample.out inception_v3.quantized.circle

       onecc -C configuration file
              Run ONE driver according to configuration section parameter

       onecc section decides whether to use each driver or not.  If the  value  is  False,
       even if the corresponding section exists, the driver won't be executed.

COPYRIGHT
       Copyright  ©  2020-2021  Samsung Electronics Co., Ltd. All Rights Reserved Licensed
       under the Apache License, Version 2.0 https://github.com/Samsung/ONE

SEE ALSO
       The full documentation for onecc is maintained as a Texinfo manual. If the info and
       onecc programs are properly installed at your site, the command

              info onecc

       should give you access to the complete manual.

onecc version 1.16.1                    August 2021                               ONECC(1)
```

**Comment by mhs4670go** [comment]
@nnfw-bot test cross-runtime-test-debug

**Comment by mhs4670go** [comment]
@nnfw-bot test cross-runtime-test-release

**Comment by mhs4670go** [comment]
@nnfw-bot test cross-aarch64-runtime-test-release

File: infra/debian/compiler/docs/onecc.1 [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+\fBpack\fR
+Package circle and metadata into nnpackage. See one-pack(1) for details.
+.SS run backend tools
+.TP
+\fBcodegen\fR
+Code generation tool. See one-codegen(1) for details.
+.SH EXAMPLES
+.SS Use command line interface
+.TP
+\fBonecc import tf --v1 -i\fR \fIinput path\fR \fB-o\fR \fIoutput path\fR \fB-I\fR \fIinput arrays\fR \fB-s\fR \fIinput shapes\fR \fB-O\fR \fIoutput arrays\fR
```
Comment (mhs4670go) [inline infra/debian/compiler/docs/onecc.1:48]: How about concatenating each word like this?
```suggestion
\fBonecc import tf --v1 -i\fR \fIinput_path\fR \fB-o\fR \fIoutput_path\fR \fB-I\fR \fIinput_arrays\fR \fB-s\fR \fIinput_ shapes\fR \fB-O\fR \fIoutput_arrays\fR
```

**Review by mhs4670go — APPROVED** [review]
LGTM

File: infra/debian/compiler/docs/onecc.1 [hunk 2]
```diff
@@ -0,0 +1,156 @@
+.\" Manpage for onecc.
+.\" Contact nnfw@samsung.com to correct errors or typos.
+.TH ONECC "1" "August 2021" "onecc version 1.16.1" "User Commands"
```
Comment (seanshpark) [inline infra/debian/compiler/docs/onecc.1:3]: Adding date and version here will let maintenance revise this in every release.
Are you going to that every release?

Comment (jyoungyun) [reply infra/debian/compiler/docs/onecc.1:3]: It is the same rule with other man pages in our project. I want to follow the existing rules.
Can you make another issue for this?

Comment (jyoungyun) [reply infra/debian/compiler/docs/onecc.1:3]: When I see the several documents, the Title is acceptable for me.

In this case, when we release new version, we need to update this document too.

https://ubuntu-mate.community/t/how-to-create-your-own-man-pages/7931

Comment (seanshpark) [reply infra/debian/compiler/docs/onecc.1:3]: > Can you make another issue for this?

Um? My question was

> Are you going to that every release?

Comment (seanshpark) [reply infra/debian/compiler/docs/onecc.1:3]:
```
man vim
man gcc
```

at least these two don't show versions. or is this file for another interface?

Comment (jyoungyun) [reply infra/debian/compiler/docs/onecc.1:3]: This file has same rules with other our man docs. I want to update it at once. So I suggest to make another issue for this.

https://github.com/Samsung/ONE/blob/438c765edee6b2b7a3d22a7879a9c438731e93b1/infra/debian/compiler/docs/one-codegen.1#L1
https://github.com/Samsung/ONE/blob/438c765edee6b2b7a3d22a7879a9c438731e93b1/infra/debian/compiler/docs/one-import-bcq.1#L1
...

For reference,
`man git`
`man cmake`
`man cat`

and so many tools have a version in man doc.

Comment (seanshpark) [reply infra/debian/compiler/docs/onecc.1:3]: I don't understand why you don't answer on my simple question `Are you going to that every release?`
Adding version/date is just a concern for the maintenance.

Adding version/date in man page documents is not my interest and I don't care itself.

I don't know technically about man pages and it's hard to read the document.

Please go on without my approval.

**Comment by seanshpark** [comment]
This PR title starts with `[one-cmds]` but is actually in `infra` folder
I used `[XXX]` in prefix of the PR title do distinghish `compiler/XXX` module.
As you are using in different meanings, this confues me.
And as I am not following any activity in current UX related task, so please do not add me as a reviewer, 
I don't know the backgrounds thus I can not decide go or no-go.

File: infra/debian/compiler/docs/onecc.1 (hunk 2 above)
Comment (jyoungyun) [reply infra/debian/compiler/docs/onecc.1:3]: > I don't understand why you don't answer on my simple question Are you going to that every release?

I thought I had already answered your question like this:

> In this case, when we release new version, we need to update this document too.

As I know, many man doc describe their version. You can see the version and date information if you scroll all the way to the end.

Thank you for review.

**Comment by jyoungyun** [comment]
> I used [XXX] in prefix of the PR title do distinghish compiler/XXX module.

I understood the rule in ONE project now. I will update the header of this issue.
And I'm sorry for adding you as reviewer. I think you might have reviewed this issue before.
But when I saw the old issue(#7182), I found that you didn't review it.
Next time I won't add you as a reviewer.

**Comment by jyoungyun** [comment]
@hseok-oh Can you review and merge this issue?

**Review by glistening — APPROVED** [review]
LGTM as a member of UX T/F

**Comment by hseok-oh** [comment]
Maybe we need version update tool for compiler like runtime version update tool `tools/release_tool/onert_version.sh`. 

BTW, I remember we decided to maintain master branch's version as next release version. So current master branch's version should 1.17.0. Did we abandon this rule?

**Review by hseok-oh — APPROVED** [review]

**Pull request merged**